SpatialGDK/Source/SpatialGDKTests/SpatialGDKEditor/SpatialGDKEditorSchemaGenerator/SpatialGDKEditorSchemaGeneratorTest.cpp

```diff
@@ -1,9 +1,859 @@
 // Copyright (c) Improbable Worlds Ltd, All Rights Reserved
 
 #include "TestDefinitions.h"
+#include "SpatialGDKEditorSchemaGenerator.h"
+#include "SpatialGDKServicesModule.h"
+#include "SchemaGenObjectStub.h"
+#include "ExpectedGeneratedSchemaFileContents.h"
+
+#include "Utils/SchemaDatabase.h"
 
 #include "CoreMinimal.h"
+#include "HAL/PlatformFilemanager.h"
+#include "Misc/FileHelper.h"
 
 #define SCHEMA_GENERATOR_TEST(TestName) \
 	TEST(SpatialGDKEditor, SchemaGenerator, TestName)
 
+namespace
+{
+const FString gSchemaOutputFolder = FPaths::Combine(FSpatialGDKServicesModule::GetSpatialOSDirectory(), TEXT("Tests/"));
```

Comment:
prefix `g`?

Reply:
`g` for global, can remove it though.

Reply:
No preference, depends on what Unreal does :D

Reply:
removed `g`